scan-external-contacts/config.yml

```diff
@@ -0,0 +1,146 @@
+  content: |
+    This tutorial walks through the steps of implementing a function to scan through the external contacts of a Genesys Cloud organization, and to process them before saving them into a file.
+
+    There are 4 types of objects that can be retrieved using the External Contacts Scanning API: External Contacts, External Organizations, Relationships, and Notes.
+
+    Go here for more information about [External Contacts Scanning](/api/rest/v2/externalcontacts/scan.html).
+
+    For this tutorial, we are going to use the Genesys Cloud SDK, and we will scan and export the External Contacts.
+    
+    The sample code is using javascript async/await (making it synchronous) to ease readibility. You can change and update it based on your needs.
+
+- title: Authentication
+  content: |
+    You will need an OAuth client using Client Credentials to authenticate. See [Create an OAuth Client](https://help.mypurecloud.com/articles/create-an-oauth-client/) for more info
+    
+    Your OAuth client (Client Credentials Grant) will need the following permissions (assigned through a role).
+    
+    To scan External Contacts, Relationships or Notes: `externalContacts:contact:view`
+    
+    To scan External Organizations: `externalContacts:externalOrganization:view`
+    
+    Update this sample with your OAuth ClientID, ClientSecret, and your Genesys Cloud region.
+
+- title: Generate an OAuth client token
+  content: |
+    In order to call the External Contacts Scanning endpoint you will need to use your OAuth client to generate a token. See [Grant - Client Credentials](/api/rest/authorization/use-client-credentials.html) for more info
+
+- title: Sample Parameters
+  content: |
+    The sample code defines a set of parameters, specific to the sample, to make it easier to modify them and to adapt them to your needs.
+
+    `maxRecordsPerPage`: the External Contacts Scan API endpoints allow to retrieve from 10 (min) to 200 (max) records per page. Default value is 100.
+
+    `minAPIRequestInterval` and `minWaitTime`: the sample code forces a pause between 2 API requests. This is to make sure that if you run the script, you will not hit an API rate limit (300 requests/minute for the client token). The sample is currently configured for more or less 2 API requests/second (500ms interval).
+    
+    You can adapt the pause duration to your needs and current use of API in your organization.
+    
+    As the scan of External Contacts (for export/save) is likely not something that has to return immediately (no real-time needs), it shouldn't be a problem to make a pause, and even to execute your code via a nightly batch.
+
+    The sample also manages a maximum number of consecutive login failures or rate limit errors (the application will exit).
+
+    It is possible to export the External Contacts to a CSV file (and to configure the field separator) or to a raw JSON file (JSON Array of External Contacts).
+
+- title: Main Function
```

Comment:
Titles should generally be unique. If there's nothing else to call these steps, maybe combine them.